.mergify.yml

```diff
@@ -16,6 +16,8 @@ pull_request_rules:
     actions:
       merge:
         method: merge
+        strict: smart+fasttrack
```

Comment:
can you say what this means and where it comes from? In https://docs.mergify.io/merge-action.html#strict-merge I don't find that

Reply:
One click further you will find the [merge action](https://docs.mergify.io/actions.html#merge-action) explained which is what the above is. `strict: true` would mean all branches are updated the moment *master* changes. `strict: smart` would mean PRs are updated *serially*, `strict: smart+fastttrack` means that plus branches already in sync with *master* can be merged right away (no re-run of already updated branches).

Reply:
Yes, sounds sensible to me. I am still unsure if there are any merges done *into* the feature branches because same as @perlpunk I would like to avoid this. But I guess we can try.

Reply:
Actually I would like to try that out in a test repository first. Can you setup one? Or simply enable the services for your fork and create a PR in your own fork from a feature branch to your master.

Reply:
Here's an example of an automatic merge https://github.com/kalikiana/openQA/pull/5 which resembles my having rebased my own branch - except Mergify did it just after master changed. I took some screenshots so you can see what it looks like as it happens:

![Screenshot 2020-10-12 at 12 23 23](https://user-images.githubusercontent.com/1204189/95736337-61268f00-0c86-11eb-96e1-5bab760a66c4.png)

---

![Screenshot 2020-10-12 at 12 23 34](https://user-images.githubusercontent.com/1204189/95736376-726f9b80-0c86-11eb-8c4a-b7258013c92a.png)

---

![Screenshot 2020-10-12 at 12 24 37](https://user-images.githubusercontent.com/1204189/95736387-77344f80-0c86-11eb-9537-33ff3149d636.png)

---

![Screenshot 2020-10-12 at 12 24 54](https://user-images.githubusercontent.com/1204189/95736410-7e5b5d80-0c86-11eb-83da-847a546a562c.png)

Reply:
hm, ok. Doesn't that run tests on circleCI all again? Could have detrimental impact during busy times but not a stopper for me.